tools/gce/linux_performance_worker_init.sh

```diff
@@ -135,6 +135,12 @@ curl -sSL https://get.rvm.io | bash -s stable --ruby
 source ~/.rvm/scripts/rvm
 gem install bundler
 
+# PHP dependencies
```

Comment:
Oh, you mean the currently running workers. No, it's not on those yet. Ok, I'll get that list from you and can figure out how to do this. I have run it on a new worker built from scratch.